lib/ThreeScaleClient.php

```diff
@@ -94,7 +107,7 @@ public function getHost() {
    */
   
      public function authorize($appId, $appKey = null, $credentials_or_service_id, $usage = null) {
-    $url = "http://" . $this->getHost() . "/transactions/authorize.xml";
+    $url = "http://" . $this->getHost() . ":" . $this->getPort() . "/transactions/authorize.xml";
```

Comment:
`https` is a good default if the library supports persistent connections. not sure how that works in PHP though. without persistence the performance is atrocious.